app/views/region/facts/_designations.html.erb

```diff
@@ -0,0 +1,48 @@
+<% designations_count = @designations_by_jurisdiction[level].count rescue 0 %>
+
+<div class="one-thirds one-thirds--no-margin">
+  <div class="fact">
+    <h2 class="fact__header"><%= level %> designations<i class="fact__external-link fa fa-external-link-square"></i></h2>
+    <div class="fact__body fact__body--lateral-padding">
+      <p class="fact__single-figure fact__single-figure--push-down">
+        <%= designations_count %>
+      </p>
+
+      <% if jurisdiction = Jurisdiction.find_by_name(level) %>
+        <div class="u-scrollable-wrapper">
+          <table class="table js-sortable-table">
+            <thead>
+              <tr class="table__header">
+                <th class="table__column table__column--with-sorting">Categories</th>
+                <th class="table__column table__column--with-sorting">Count</th>
+              </tr>
+            </thead>
+
+            <tbody>
+              <% @region.protected_areas_per_designation(jurisdiction).each do |row| %>
+                <tr class="table__row">
+                  <td class="table__label table__label--link">
+                    <%= link_to row["designation_name"], search_url(region: @region.name, designation: row["designation_name"], main: :region) %>
+                  </td>
+                  <td class="table__value">
+                    <%= row["count"] %>
+                  </td>
+                </tr>
+              <% end %>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      <% end %>
+
+      <% if designations_count > 10 %>
+        <div class="space-between">
+          <div class="fact__link">
+          </div>
+          <div class="fact__link">
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
Do you think we could use a shared partial between regions and countries? Or would you prefer doing this along with all the others in a later PR?

Reply:
There's a slight difference; `@country` vs `@region` but we can abstract this out. I would say let's merge it for now and then have a card to do it later